src/PowerShellEditorServices/Session/Host/EditorServicesPSHost.cs

```diff
@@ -87,6 +87,28 @@ internal ConsoleColorProxy(EditorServicesPSHostUserInterface hostUserInterface)
                 _hostUserInterface = hostUserInterface;
             }
 
+            /// <summary>
+            /// The Accent Color for Formatting
+            /// </summary>
+            public ConsoleColor FormatAccentColor
+            {
+                get
+                { return _hostUserInterface.FormatAccentColor; }
```

Comment:
NIT
```suggestion
                {
                    return _hostUserInterface.FormatAccentColor;
                }
```

Reply:
This is your code @SteveL-MSFT... (see the commit)